rasa/nlu/model.py

```diff
@@ -381,4 +381,10 @@ def parse(
 
         output = self.default_output_attributes()
         output.update(message.as_dict(only_output_properties=only_output_properties))
+
+        if self.model_metadata:
+            output.update(
+                {"metadata": {"model": self.model_metadata.get("trained_at")}}
```

Comment:
The names are not very clear in my opinion. `model` does not seem to be the same thing as `trained_at`.

Reply:
Yes - but you build the model name from the timestamp that it was trained on.